docusaurus/website/src/theme/tracking/index.ts

```diff
@@ -16,7 +16,7 @@ const isRudderstackSetup = (config: RudderStackTrackingConfig) =>
 // Enqueue all rudderstack requests until it is loaded and consent is granted
 let rudderstack = {};
 let rudderQueue = [];
-const rudderMethods = ["page", "track", "identify", "reset"];
+const rudderMethods = ['page', 'track', 'identify', 'reset'];
```

Comment:
is there a prettier config or something we should collectively use to have the same styling? otherwise it feels like we will have changes that are just styling back and forth depending on who commited them / how their IDE is setup.

Reply:
There is a prettierrc config file in this repo. The reason for this addition in this PR is due to added a git hook recently to make sure files are fixed on commit.